gp/fitness.py

```
@@ -0,0 +1,48 @@
+from .primitives import pset
+
+
+def evaluate(individual, data):
+    func = gp.compile(expr=individual, pset=pset)
+
+    balance = 10000.0
+    equity = balance
+    max_dd = 0.0
+    trades = 0
+
+    for i, row in data.iterrows():
```

Ruff (0.11.9)

Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)